closure/goog/color/alpha.js

```diff
@@ -68,21 +68,26 @@ goog.color.alpha.hexToRgbaStyle = function(hexColor) {
 };
 
 
+function extractColor(startIndex, endIndex) {
```

Comment:
- This isn't a goog.module, you don't want to introduce a global name here.
- This is missing proper jsdoc and type declarations
- It would be better to call a private method with parameters, rather than to return a function expression.  The former is more regular and more likely to be optimized by VMs and the Closure Compiler.